cookbook/vis_mpl_seismic.py

```diff
@@ -0,0 +1,37 @@
+"""
+Seismic: seismic plotting utilities uses Obspy for SEGY reading
+"""
+from fatiando.vis import mpl
+from obspy.segy import segy
+import urllib
+import numpy as np
+
+# fetch sample SEGY data, near-offset marmousi data
+url = "http://dl.dropboxusercontent.com/" \
+      "s/i287ci4ww3w7gdt/marmousi_nearoffset.segy"
+urllib.urlretrieve(url, 'marmousi_nearoffset.segy')
+segyfile = segy.readSEGY('marmousi_nearoffset.segy')
```

Comment:
Could you add a comment above this line saying: "We'll use the obspy library to load the SEGY data"